hub/core/chunk_engine.py

```diff
@@ -774,18 +802,20 @@ def numpy(
             for global_sample_index in index.values[0].indices(length):
                 chunk_ids = enc[global_sample_index]
                 if not self._is_tiled_sample(global_sample_index):
-                    chunk = self.get_chunk_from_chunk_id(chunk_ids[0])
                     local_sample_index = enc.translate_index_relative_to_chunks(
                         global_sample_index
                     )
                     if self.compression in VIDEO_COMPRESSIONS:
+                        chunk, stream = self.get_url_or_chunk(chunk_ids[0])
                         sample = chunk.read_sample(
                             local_sample_index,
                             sub_index=index.values[1].value  # type: ignore
```

Comment:
unrelated to this PR, but would prefer this if-else to be out of read_sample for readability